cost/google/cud_recommendations/google_committed_use_discount_recommendations.pt

```diff
@@ -0,0 +1,371 @@
+name "Google Committed Use Discount Recommender"
```

Comment:
Does this policy require changelog updates? Can't see it in this pull request but please correct me if wrong